n_atof.c

```diff
@@ -257,7 +257,7 @@ char **endPtr;              /* If non-NULL, store terminating character's
         case 5:
             p10 = 1.0e32;
             break;
-#ifndef NOTE_FLOAT
+#ifndef NOTE_LOWMEM
```

Comment:
I would like to yeah. In theory there could be users depending on this macro. Should we change the define to `NOTE_C_LOWMEM`, but also define `NOTE_LOWMEM` if `NOTE_C_LOWMEM` is defined for backwards compatibility?

The same goes for `NOTE_FLOAT`. I opted to eliminate it entirely, but there could be someone out there using it in their code.

Reply:
You're right. I think we should change it to `NOTE_C_LOWMEM`, and also declare `NOTE_LOWMEM`. There should probably be a deprecation message before it is `#define`'d, in case it was declared by the build system.